openlibrary/i18n/fr/messages.po

```diff
@@ -4588,18 +4588,18 @@ msgstr "Consultez WorldCat pour une édition près de chez vous"
 #~ msgid ""
 #~ "This user has chosen to publicly "
 #~ "share the books they are <a "
-#~ "href=\"$(page.key)/books/currently-reading\">currently "
-#~ "reading</a>, <a href=\"$(page.key)/books/already-"
+#~ "href=\"%(username)s/books/currently-reading\">currently "
+#~ "reading</a>, <a href=\"%(username)s/books/already-"
 #~ "read\">have already read</a>, and <a "
-#~ "href=\"$(page.key)/books/want-to-read\">want to "
+#~ "href=\"%(username)s/books/want-to-read\">want to "
 #~ "read</a>!"
 #~ msgstr ""
 #~ "Cet utilisateur a choisi de partager "
 #~ "publiquement les livres qu'il est "
-#~ "<ahref=\"$(page.key)/books/currently\"> en cours de"
-#~ " lecture </a>, <ahref=\"$(page.key)/books/already-"
+#~ "<ahref=\"%(username)s/books/currently\"> en cours de"
+#~ " lecture </a>, <ahref=\"%(username)s/books/already-"
 #~ "read\"> ont déjà lu </a>, et "
-#~ "<ahref=\"$(page.key)/books/want-to-read\"> voulez"
+#~ "<ahref=\"%(username)s/books/want-to-read\"> voulez"
 #~ " lire </a>!"
```

Comment:
This entire block can just be deleted; the `#~` denote comments; these are only included as backups, but the real string is already in here.